app/models/job_template.rb

```diff
@@ -87,6 +88,11 @@ def effective_user
     super || (build_effective_user.tap(&:set_defaults))
   end
 
+  def generate_description_format
+    format = description_format || template_inputs.map(&:name).map { |name| %Q(#{name}="%{#{name}}") }.join(' ')
+    format == '' ? '%{job_name}' : format
```

Comment:
The `%{job_name}` should be part of the generate format even when the format is not empty (when there is some input). And maybe in format of `Run %{job_name} with inputs…`